webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -184,19 +185,53 @@ let GitBlameInfoModal = React.createClass({
     getBranch() {
         try {
             return this.props.gitBlameWithUsage.branch.name;
-        } catch(e) {
+        } catch (e) {
             return " - ";
         }
     },
 
     getBranchScreenshots() {
         try {
             return this.props.gitBlameWithUsage.branch.screenshots;
-        } catch(e) {
+        } catch (e) {
             return [];
         }
     },
 
+    getParamsForLinks() {
+        return {
+            textUnitName: this.props.textUnit.getName(),
+            assetPath: this.props.textUnit.getAssetPath(),
+            textUsageNamePrefix: this.props.textUnit.getName().split("_")[0],
+            textUnitNameWithoutPluralForm: this.getTextUnitNameWithoutPluralForm()
+        };
+    },
+
+    getTextUnitNameWithoutPluralForm() {
+        let prefix = this.props.textUnit.getName();
+
+        if (this.props.textUnit.getPluralForm() != null) {
+            // remove plural form. remove optionally a space for this to work with different types but that break
+            // type that don't use it and have a text unit name that ends with space.
+            prefix = prefix.replace(/\s?_(zero|one|two|few|many|other)$/, "");
+        }
+
+        return prefix;
+    },
+
+    renderLink(urlTemplate, urlComponentTemplate, labelTemplate, params) {
+        let url = LinkHelper.renderUrl(urlTemplate, urlComponentTemplate, params);
+        let label = LinkHelper.renderTemplate(labelTemplate, params);
+        return <a href={url}>{label}</a>;
+    },
+
+    getThirdPartyLink() {
+        let params = this.getParamsForLinks();
+        let url = LinkHelper.renderUrl(this.getThirdPartyUrlTemplate(), this.getThirdPartyUrlComponentTemplate(), params);
+        let label = LinkHelper.renderTemplate(this.getThirdPartyLabelTemplate(), params);
+        return <a href={url}>{label}</a>;
```

Comment:
you can just reuse the logic of `renderLink` to create the link: `return this.renderLink(this.getThirdPartyUrlTemplate(), this.getThirdPartyUrlComponentTemplate(), this.getThirdPartyLabelTemplate(), params)`

Reply:
:D introduced the function for this... Thanks for catching!